src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -123,6 +124,7 @@ class ConvertThread : public render::TaskDelegate {
                     m_image->height()));
 
     render::Render render;
+    render.setNewBlend(Preferences::instance().experimental.newBlend());
```

Comment:
It would be better if this `newBlend` value is passed as an argument in this `run()` function, because this is executed in a background thread (not the UI main thread), and I'm not sure if the `Preferences` class is thread-safe.